test/src/test/java/jenkins/security/s2m/DefaultFilePathFilterTest.java

```diff
@@ -64,15 +65,9 @@ public void setUp() {
         assertEquals("hello", s.getRootPath().act(new LocalCallable(forward)));
         FilePath reverse = new FilePath(new File(r.jenkins.root, "reverse"));
         assertFalse(reverse.exists());
-        try {
-            s.getChannel().call(new ReverseCallable(reverse));
-            fail("should have failed");
-        } catch (Exception x) {
-            // good
-
-            // make sure that the stack trace contains the call site info to help assist diagnosis
-            assertThat(Functions.printThrowable(x), containsString(DefaultFilePathFilterTest.class.getName()+".remotePath"));
-        }
+        IOException x = assertThrows(IOException.class, () -> s.getChannel().call(new ReverseCallable(reverse)));
```

Comment:
Should `ReverseCallable` be changed accordingly? It declares that it throws `Exception`, so `#call` does too.

Reply:
> Should `ReverseCallable` be changed accordingly?

This bound can't be tightened, because `#call` could throw either `IOException` or `InterruptedException`.